skin/src/main/java/org/researchstack/skin/AppPrefs.java

```diff
@@ -24,11 +24,16 @@
         prefs = PreferenceManager.getDefaultSharedPreferences(context);
     }
 
-    public static synchronized AppPrefs getInstance(Context context)
+    public static void init(Context context) {
```

Comment:
As long as init is called from the main thread, i don't think we would hit thread issues.  SharedPrefs is thread-safe